### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by thegtproject** [p-groarke/wsay#4]
Build currently fails on Windows 10. 

```
c:\dev\repos\wsay\build>cmake -DCMAKE_TOOLCHAIN_FILE=C:/dev/tools/vcpkg/scripts/buildsystems/vcpkg.cmake -DBUILD_TESTING=Off ..
-- Building for: Visual Studio 16 2019
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
-- The CXX compiler identification is MSVC 19.28.29337.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Creating directories for 'wil-populate'
  Performing download step (git clone) for 'wil-populate'
  Cloning into 'wil-src'...
  Host key verification failed.
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights
  and the repository exists.
  Cloning into 'wil-src'...
  Host key verification failed.
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights
  and the repository exists.
  Cloning into 'wil-src'...
  Host key verification failed.
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights
  and the repository exists.
  -- Had to git clone more than once:
  CMake Error at wil-subbuild/wil-populate-prefix/tmp/wil-populate-gitclone.cmake:31 (message):
            3 times.
    Failed to clone repository: 'git@github.com:p-groarke/wil.git'


C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(238,5): error MSB8066: Custom build for 'C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-mkdir.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-download.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-update.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-patch.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-configure.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-build.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-install.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b5eecd10a1da9181ae3c64e5ab768bb9\wil-populate-test.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\fe3737c709904e235664aa0461db5ce0\wil-populate-complete.rule;C:\dev\repos\wsay\build\_deps\wil-subbuild\CMakeFiles\b625d97caad1fb9f2aea7f050cea2ff8\wil-populate.rule' exited with code 1. [C:\dev\repos\wsay\build\_deps\wil-subbuild\wil-populate.vcxproj]

CMake Error at C:/Program Files/CMake/share/cmake-3.18/Modules/FetchContent.cmake:987 (message):
  Build step for wil failed: 1
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.18/Modules/FetchContent.cmake:1082:EVAL:2 (__FetchContent_directPopulate)
  C:/Program Files/CMake/share/cmake-3.18/Modules/FetchContent.cmake:1082 (cmake_language)
  C:/Program Files/CMake/share/cmake-3.18/Modules/FetchContent.cmake:1125 (FetchContent_Populate)
  CMakeLists.txt:76 (FetchContent_MakeAvailable)


-- Configuring incomplete, errors occurred!
See also "C:/dev/repos/wsay/build/CMakeFiles/CMakeOutput.log".
```

**Comment by p-groarke** [comment]
Hey Brandon, thx for reporting this. I'll get it fixed asap, in the meantime this should get it to build. Let me know if you have other issues.

In cmakelists.txt, at line ~71, uncomment the wil repo and remove the old one :
```cmake
# Before
FetchContent_Declare(wil
	GIT_REPOSITORY git@github.com:p-groarke/wil.git
	GIT_TAG cmake
	# GIT_REPOSITORY https://github.com/microsoft/wil.git
)

# After
FetchContent_Declare(wil
	GIT_REPOSITORY https://github.com/microsoft/wil.git
)
```

That should fix it.

**Comment by p-groarke** [comment]
Fixed by 64c58c2497dc6647f708ba69c1ff073f75d39d55
